doc/user/download_backup.md

```diff
@@ -0,0 +1,49 @@
+# Download Backup
+
+The backups that etcd operator saves are [etcd snapshots](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/recovery.md).
+This document talks about the ways to download these backup/snapshot files.
+
+## Backup Service
+
+If etcd cluster with backup is still running, there is a backup service.
+It's named as `${CLUSTER_NAME}-backup-sidecar`. For example:
+```
+$ kubectl get svc
+etcd-cluster-backup-sidecar   10.39.243.229   <none>        19999/TCP           4m
+```
+
+Given the etcd version of the cluster, you can download backup via HTTP endpoint
+`http://${CLUSTER_NAME}-backup-sidecar:19999/backup?etcdVersion=${ETCD_VERSION}`. For example:
+```
+$ curl "http://etcd-cluster-backup-sidecar:19999/backup?etcdVersion=v3.1.0" -o v3.1.0_etcd.backup
+```
+
+It's up to users to decide how to access the backup service, e.g. using ingress.
+
+## S3
+
+If etcd cluster isn't running, e.g. k8s is down, and backup storage type is "S3",
```

Comment:
If the backup service pod is running OK, we suggest to get backups from the backup service.

However, in extreme cases like when Kubernetes is down, and the backup service is not running, the backup cannot be retrieved from backup service.

Users have to get the backup directly from S3.